internal/oci/runtime_oci.go

```diff
@@ -198,6 +198,14 @@ func (r *runtimeOCI) CreateContainer(ctx context.Context, c *Container, cgroupPa
 	}(); err != nil {
 		return err
 	}
+
+	defer func() {
+		if retErr != nil {
+			if err := os.Remove(c.logPath); err != nil {
+				log.Warnf(ctx, "Failed to remove log path %s after failed to create container: %v", c.logPath, err)
```

Comment:
nit
```suggestion
				log.Warnf(ctx, "Failed to remove log path %s after failure to create container: %v", c.logPath, err)
```
or perhaps "failing"